src/components/Navigation/Ui/ComponentButtons.js

```diff
@@ -59,14 +61,14 @@ export default class ComponentButtons extends Component {
   }
 
   render() {
-    const { definition, currentComponent, readOnly, hasChange, hideVersionSelector } = this.props
+    const { definition, curations, currentComponent, readOnly, hasChange, hideVersionSelector } = this.props
     const component = EntitySpec.fromCoordinates(currentComponent)
     const isSourceComponent = this.isSourceComponent(component)
     const scores = Definition.computeScores(definition)
     const isDefinitionEmpty = Definition.isDefinitionEmpty(definition)
     const isSourceEmpty = Definition.isSourceEmpty(definition)
-    const isCurated = Definition.isCurated(definition)
-    const hasPendingCurations = Definition.hasPendingCurations(definition)
+    const isCurated = Curation.isCurated(curations)
+    const hasPendingCurations = Curation.hasPendingCurations(curations)
```

Comment:
`curations` here is undefined, still needs to be wired up

Reply:
To do that , `curations` should be returned by invoking the /curation API for each component in the list.
Is it really worthwhile or maybe we could consider to have these data in the /definition API response? 
Those are only needed to show the status of the curation (isCurated, hasPendingCurations).

Reply:
I agree. not worthwile, let's back this out